mkt/webapps/tasks.py

```diff
@@ -715,6 +715,10 @@ def _get_trending(app_id):
         return {}
 
     def _score(week1, week3):
+        # If last week app installs are < 100, this app isn't trending.
+        if week1 < 100.0:
```

Comment:
Ok, if there's some comments where it came from and that we can tweak it, that's cool. Just wasn't sure myself.